### DISCUSSION TITLE
An option to toggle off estimate

### DISCUSSION THREAD

**Discussion by ivilla-clickup** [intellibar/main#7] [Ideas]
I would love the ability to toggle this off. It sometimes gets distracting. Having a toggle in the settings or even an option to still preview some of the data in the settings would be helpful.

![CleanShot 2023-08-09 at 19 27 26@2x](https://github.com/intellibar/main/assets/111552059/2ee25a17-0d35-4722-bd41-5cc7eae4daaf)

**Comment by erusev** [comment]
Makes sense, I think we can do that, thanks for the suggestion.
